sources/packages/api/src/services/institution-location/institution-location.models.ts

```diff
@@ -1,4 +1,26 @@
-export interface LocationWithDesigationStatus {
+export interface LocationWithDesignationStatus {
+  id?: number;
   locationName: string;
   isDesignated: boolean;
+  locationAddress?: AddressData;
+  institutionCode?: string;
+  primaryContact?: PrimaryContact;
+}
+interface PrimaryContact {
```

Comment:
I would avoid recreating the interfaces unless we need something different at the layer.
The interface `LocationWithDesignationStatus` can reuse some types from the entity model if they are really using them entirely, which is the case here.
Please note that the opposite would not be valid, for instance, changing the entity layer to accommodate some change that it would be needed in the service layer. Please let me know if we need some clarification on this. 😉